coderd/database/dbauthz/dbauthz.go

```diff
@@ -1792,6 +1792,10 @@ func (q *querier) GetProvisionerJobByID(ctx context.Context, id uuid.UUID) (data
 }
 
 func (q *querier) GetProvisionerJobTimingsByJobID(ctx context.Context, jobID uuid.UUID) ([]database.ProvisionerJobTiming, error) {
+	_, err := q.GetProvisionerJobByID(ctx, jobID)
```

Comment:
This approach raised my eyebrow, and I was going to suggest not doing it this way but I see we have precedent for this in `GetProvisionerJobByID` itself. Please add a comment explaining this.

Reply:
I'm sure we use a similar pattern for workspace builds or templates.